### PR TITLE
BD-3081 Update export troubleshooting

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-3081**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md [hunk 1]
```diff
@@ -24,12 +24,22 @@ If you are using a **Braze S3 bucket**, this could happen because:
...
 This could also be caused by Braze no longer having access to the S3 bucket you are downloading the data to. Make sure you've updated your S3 credentials using these steps.
 
-#### "Looks like the file doesn't exist anymore, please check to make sure nothing is deleting objects from your bucket"
+### "Looks like the file doesn't exist anymore, please check to make sure nothing is deleting objects from your bucket"
 
 There may be a slight lag between when Braze's email with the export gets sent, and when S3 is actually ready to serve the object. If you see this error, wait a few minutes before trying again.
 
+### Apostrophes added to fields
+
+Braze will automatically prepend an appostrophe to a field in the CSV output if the field begins with any of the following characters:
```
Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md:38]:
```suggestion
Braze will automatically prepend an appostrophe to a field in the CSV export if the field begins with any of the following characters:
```

File: _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md [hunk 2]
```diff
@@ -24,12 +24,22 @@ If you are using a **Braze S3 bucket**, this could happen because:
...
 This could also be caused by Braze no longer having access to the S3 bucket you are downloading the data to. Make sure you've updated your S3 credentials using these steps.
 
-#### "Looks like the file doesn't exist anymore, please check to make sure nothing is deleting objects from your bucket"
+### "Looks like the file doesn't exist anymore, please check to make sure nothing is deleting objects from your bucket"
 
 There may be a slight lag between when Braze's email with the export gets sent, and when S3 is actually ready to serve the object. If you see this error, wait a few minutes before trying again.
 
+### Apostrophes added to fields
+
+Braze will automatically prepend an appostrophe to a field in the CSV export if the field begins with any of the following characters:
```
Comment (calebsevern) [inline _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md:38]: appostrophe -> apostrophe

Comment (rachel-feinberg) [inline _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md:38]:
```suggestion
Braze will automatically prepend an apostrophe to a field in the CSV export if the field begins with any of the following characters:
```

**Review by calebsevern — APPROVED** [review]

File: _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md [hunk 3]
```diff
@@ -24,12 +24,22 @@ If you are using a **Braze S3 bucket**, this could happen because:
...
+### Apostrophes added to fields
+
+Braze will automatically prepend an apostrophe to a field in the CSV export if the field begins with any of the following characters:
+
+- -
+- =
+- +
+- @
+
+For example, the field "-1943" will be exported as "'-1943". This does not apply to JSON exports, such as those returned by the [`/users/export/segment`]({{site.baseurl}}/api/endpoints/export/user_data/post_users_segment/) endpoint.
```
Comment (lydia-xie) [inline _docs/_user_guide/data_and_analytics/export_braze_data/export_troubleshooting.md:45]:
```suggestion
Braze will automatically prepend an apostrophe to a field in the CSV export if the field begins with any of the following characters:

- -
- =
- +
- @

For example, the field "-1943" will be exported as "'-1943". This does not apply to JSON exports, such as those returned by the [`/users/export/segment` endpoint]({{site.baseurl}}/api/endpoints/export/user_data/post_users_segment/).
```

**Review by lydia-xie — APPROVED** [review]
lgtm!

**Comment by lydia-xie** [comment]
@cla-bot check

**Pull request merged**